.github/workflows/lint.yml

```diff
@@ -120,6 +120,17 @@ jobs:
         run: |
           find doc/ -name '*.py' | xargs flake8
           find tests/ -name '*.py' | xargs flake8
+        
+      - name: Check integration test IDs
+        run: |
+          FMFFILES=$(find tests/ -name 'main.fmf' ! -path "tests/tests/main.fmf" ! -path "tests/tests/tier0/main.fmf")
```

Comment:
Hmm, I think it can be further simplified:

```
run: |
  MISSING_ID_FILES=$(find tests/tests/*/*/ -name main.fmf -exec grep -L '^id:' {} \;)

  if [ -n "$MISSING_ID_FILES" ]; then
      echo "Integration test ID is missing in: $MISSING_ID_FILES"
      exit 1
  fi
```